src/Messaging/Configuration/SubscriptionManagerOptions.cs

```diff
@@ -6,15 +6,13 @@ namespace CodeMonkeys.Messaging
 {
     public class SubscriptionManagerOptions : Options
     {
-        private TimeSpan _flushSubscriptionsPeriod = TimeSpan.FromSeconds(10);
-
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '10' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)